qstash/api/events/list.mdx

```diff
@@ -65,11 +65,47 @@ The name of the endpoint if this message was sent through a topic
 </ResponseField>
 
 <RequestExample>
-```sh
+```sh curl
 curl https://qstash.upstash.io/v2/events \
   -H "Authorization: Bearer <token>"
 ```
 
+```javascript Node
+fetch('https://qstash.upstash.io/v2/events', {
+  headers: {
+    'Authorization': 'Bearer <token>'
+  }
+});
+```
```

Comment:
To make the examples consistent with each other, I think we can do this. Same applies to other node.js snippets
```suggestion
```javascript Node
const response = await fetch('https://qstash.upstash.io/v2/events', {
  headers: {
    'Authorization': 'Bearer <token>'
  }
});
```
```